src/NodaTime/CalendarSystem.cs

```diff
@@ -247,7 +253,28 @@ internal static CalendarSystem ForOrdinalUncached([Trusted] CalendarOrdinal ordi
             Preconditions.CheckArgumentRange(nameof(monthNumbering), (int) monthNumbering, 1, 2);
             return HebrewCalendars.ByMonthNumbering[((int) monthNumbering) - 1];
         }
-        
+
+        /// <summary>
+        /// Returns the Wondrous (Badí') calendar, as described at https://en.wikipedia.org/wiki/Badi_calendar. 
+        /// This is a purely solar calendar with years starting at the vernal equinox.
+        /// </summary>
+        /// <remarks>
+        /// <para>The Wondrous calendar was developed and defined by the founders of the Bahá'í Faith in the mid to late
+        /// 1800's A.D. The first year in the calendar coincides with 1844 A.D. Years are labeled "B.E." for Bahá'í Era.</para>
+        /// <para>A year consists of 19 months, each with 19 days. Each day starts at sunset. Years are grouped into sets
+        /// of 19 "Unities" (Váḥid) and 19 Unities make up 1 "All Things" (Kull-i-Shay’).</para>
+        /// <para>A period of days (usually 4 or 5, called Ayyám-i-Há) occurs between the 18th and 19th months. The length of this 
+        /// period of intercalary days is solely determined by the pre-calculated time of the following vernal equinox. 
+        /// The vernal equinox is a momentary point in time, so the "date" of the equinox is determined by the day (beginning 
+        /// at sunset) in effect in Tehran, Iran at the moment of the equinox. These times can be calculated, but published
+        /// tables are used in this implementation keep it simple!</para>
+        /// </remarks>
+        /// <returns>The Wondrous calendar system.</returns>
+        [NotNull] public static CalendarSystem GetWondrousCalendar()
```

Comment:
Not sure why a method is needed here - just make it a property? (Starting from 2.0, most calendar systems are accessible via properties. We only have methods where there's parameterization.)

Reply:
(I see there's a property already, so let's just remove the method.)

Reply:
This is a case of monkey see, monkey do. It was trying to replicate what was done for other Calendars. Please adjust as desired.